ts/Core/Chart/ChartDefaults.ts

```diff
@@ -492,7 +492,7 @@ const ChartDefaults: ChartOptions = {
      *
      * @type       {string}
      * @default    {highcharts} undefined
```

Comment:
I've removed the default values for the deprecated options so that the `setZoomOptions` function is much shorter. Now, `chart.options.zoomType` is set only when the users set it (it is not always taken from `ChartDefaults`).